Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -712,10 +712,10 @@ void CFrame::StartGame(const std::string& filename)
     DoFullscreen(false);
     // Destroy the renderer frame when not rendering to main
     if (!SConfig::GetInstance().bRenderToMain)
-      m_RenderFrame->Destroy();
-    m_RenderFrame = nullptr;
-    m_RenderParent = nullptr;
-    m_bGameLoading = false;
+      m_render_frame->Destroy();
+    m_render_frame = nullptr;
```

Comment:
Could you put a newline after the `Destroy` so it's easier to see what the `if` includes?